docs/os-ref.md

```diff
@@ -4,615 +4,546 @@ title: aragonOS reference documentation
-addresses and bytes32 (in the case of comparing hashes, losing 2 bytes of precision
-shouldn't be a dealbreaker if the hash algorithm is secure). The only problem is
-when
-- **Argument ID** (`uint8`): Determines how the comparison value is fetched. From
-0 to 200 it refers to the argument index number passed to the role. After 200, there
-are some *special Argument IDs*:
-	- `BLOCK_NUMBER_PARAM_ID` (`id = 200`): Sets comparison value to the block number
-	at the time of execution. This allows for setting up timelocks depending
-	on blocks.
-	- `TIMESTAMP_PARAM_ID` (`id = 201`): Sets comparison value to the timestamp of the
-	current block at the time of execution. This allows for setting up timelocks
-	on time.
-	- `SENDER_PARAM_ID` (`id = 202`): Sets comparison value to the sender of the call.
-	(Currently useless because of [this issue]())
-	- `ORACLE_PARAM_ID` (`id = 203`): Checks with an oracle at the address in the
-	*argument value* and returns whether it returned true or false (no comparison with arg).
-	- `LOGIC_OP_PARAM_ID` (`id = 204`): Evaluates a logical operation and returns
-	true or false depending on its result (no comparison with arg).
-	- `PARAM_VALUE_PARAM_ID` (`id = 205`): Uses value as return. Commonly used with
-	the `RET` operation, to just return a value. If the value in the param is greater
-	than 0, it will evaluate to true, otherwise it will return false.
-- **Operation type** (`uint8`): Determines what operation is made to compare the
-value fetched using the argument ID or the argument value. For all comparisons,
-both values are compared in the following order `args[param.id] <param.op> param.value`.
-Therefore for a greater than operation, with a `param = {id: 0, op: Op.GT, value: 10}`,
-it will interpret whether the argument 0 is greater than 10. The implemented
-operation types are:
-	- None (`Op.NONE`): Always evaluates to `false`, regardless of parameter or arguments.
-	- Equals (`Op.EQ`): Evaluates to true if every byte matches between `args[param.id]` and
-	`param.value`.
-	- Not equals (`Op.NEQ`): Evaluates to true if any byte doesn't match.
-	- Greater than (`Op.GT`): Evaluates to true if `args[param.id] > param.value`.
-	- Less than (`Op.LT`): Evaluates to true if `args[param.id] < param.value`.
-	- Greater than or equal (`Op.GTE`): Evaluates to true if `args[param.id] >= param.value`.
-	- Less than or equal (`Op.LTE`): Evaluates to true if `args[param.id] <= param.value`.
-	- Return (`Op.RET`): Evaluates to true if `args[param.id]` is greater than one.
-	Used with `PARAM_VALUE_PARAM_ID`, it makes `args[param.id] = param.value`, so it
-	returns the parameter associated value.
-
-While also representing an operation, when the id is `LOGIC_OP_PARAM_ID`, only the
-ops below are valid. These operations use the parameter's value to point to other
-parameters index in the parameter array. These values are encoded as `uint32`
-numbers, left-shifted 32 bits to the left each (example: for example, an op that
```

Comment:
instead of "left-shifted 32 bits to the left each" how about "each left-shifted 32 bits"?

Reply:
Fixed in 63c5542